spec/lib/ordering_helper_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+def stub_ordering_helper(helper, eq_model1, eq_model3)
+  allow(helper).to receive(:ordering).and_return(3)
+  allow(helper).to receive(:predecessor).and_return(eq_model1)
+  allow(helper).to receive(:successor).and_return(eq_model3)
+  allow(helper).to receive(:rel_predecessor).with(3).and_return(1)
+  allow(helper).to receive(:rel_successor).with(3).and_return(5)
+  allow(helper).to receive(:cat_first).and_return(1)
+  allow(helper).to receive(:cat_last).and_return(5)
+  allow(helper).to receive(:successors).and_return([eq_model3])
+  allow(helper).to receive(:rel_predecessor).with(5).and_return(3)
+end
```

Comment:
you shouldn't be adding anything to spec_helper -- you should also just stub out these things as necessary in your tests inside the tests themselves.

perhaps a better approach is stubbing the original scope out on EquipmentModel to return an appropriate array?